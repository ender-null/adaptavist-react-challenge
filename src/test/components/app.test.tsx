import React from "react";
import { render } from "@testing-library/react";
import App from "../../components/app";

describe("Item", () => {
  test("should render the component", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
