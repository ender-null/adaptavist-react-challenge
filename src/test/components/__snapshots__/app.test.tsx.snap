// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item should render the component 1`] = `
<DocumentFragment>
  <div
    class="app"
  >
    <h1>
      Adaptavist React Challenge
    </h1>
    <textarea
      class="textarea"
      placeholder="Write here"
    />
    <div
      class="sorting"
    >
      <button
        class="selected"
      >
        Alphabetical
      </button>
      <button
        class="unselected"
      >
        Numerical
      </button>
    </div>
    <ul
      class="list"
    />
  </div>
</DocumentFragment>
`;
